wheels/build/wheels.yml

```diff
@@ -62,6 +62,20 @@ packages:
         prebuild:
             wheel: for f in sqlite3.h sqlite3.c sqlite3ext.h; do cp ${SRC_ROOT_1}/$f $f || return 1; done && echo 'include_dirs=.' >>${SRC_ROOT_0}/setup.cfg
         build_args: build_static bdist_wheel
+    python-ldap:
+        version: 2.4.27
+        src:
+            - http://mirror.eu.oneandone.net/software/openldap/openldap-release/openldap-2.4.44.tgz
+            - https://pypi.python.org/packages/fc/99/9eed836fe4d916792994838df125da9c25c5f7c31abfbf6f0ab076e5f419/python-ldap-2.4.27.tar.gz
```

Comment:
By looking at the `pysqlite` recipe, it *seems* that if you omit this, then `$SRC_ROOT_0` would be python-ldap and `$SRC_ROOT_1` would be openldap.